integration/src/main/kotlin/kotlinx/benchmark/integration/ProjectBuilder.kt

```diff
@@ -23,7 +29,10 @@ benchmark {
         ${targets.flatMap { it.value.lines(it.key) }.joinToString("\n        ")}
     }
 }
-            """.trimIndent()
+kotlin {
```

Comment:
WDYT about moving up the `kotlin {}` section? As we documented in the README, the step for adding Kotlin targets precedes the steps for configuring benchmarks.

Reply:
It is definitely more appropriate. Moved in the latest commit